README-template.md

```diff
@@ -88,7 +88,7 @@ To add a dependency on one of the connectors using Maven, use the following:
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>bigquery-connector</artifactId>
-      <version>${next-bigquery-connector-release-tag}</version>
+      <version>hadoop2-1.2.0</version>
```

Comment:
Why are we hardcoding the value here?

Reply:
This is the version for BQ connector. we have not released this in a while and probably will not. We have removed this from master for e.g. We can add this back if we plan to add automation for Hive BQ connector release.